repo/format/format_provider.go

```diff
@@ -99,11 +100,18 @@ func NewFormattingOptionsProvider(f *ContentFormat, formatBytes []byte) (Provide
 		return nil, errors.Wrap(err, "unable to create hash")
 	}
 
-	e, err := encryption.CreateEncryptor(f)
+	encryptionEncryptor, err := encryption.CreateEncryptor(f)
 	if err != nil {
 		return nil, errors.Wrap(err, "unable to create encryptor")
 	}
 
+	eccEncryptor, err := ecc.CreateEncryptor(f)
```

Comment:
add a comment saying this can and in most cases will be nil, since it's not quite obvious.

Reply:
Added. For some reason it does not show here.